static/js/compose.js

```diff
@@ -106,16 +106,27 @@ function update_stream_button(btn_text, title) {
     $("#left_bar_compose_stream_button_big").prop("title", title);
 }
 
-exports.update_stream_button_for_private = function () {
-    var text = i18n.t("New stream message");
-    var title = text + " (c)";
-    update_stream_button(text, title);
+function update_conversation_button(btn_text, title) {
+    $("#left_bar_compose_private_button_big").text(btn_text);
+    $("#left_bar_compose_private_button_big").prop("title", title);
+}
+
+exports.update_closed_compose_button_for_private = function () {
+    var textStream = i18n.t("New stream message");
+    var titleStream = textStream + " (c)";
+    var textConv = i18n.t("New conversation");
+    var titleConv = textConv + " (c)";
```

Comment:
We don't use camelCase for variable names in Zulip use underscores instead.  We also don't use abbreviations -- just spell it out as `text_conversation` (or even better, `conversation_text`)